From e05687e0a6b52cbe76c4f86ba959898d71c7b5be Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 15 Dec 2016 11:15:07 +0800
Subject: [PATCH] arm: dts: rockchip: switch to vpu-service in rk3288 linux
 boards

We decide to back to old vcodec driver until getting a better v4l2 solution

Change-Id: Iec3f5b3aba86866a0b0fc7aab5b6c2b28bc268d9
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-fennec.dts   |  6 +++++-
 arch/arm/boot/dts/rk3288-firefly.dtsi |  6 +++++-
 arch/arm/boot/dts/rk3288-miniarm.dts  |  8 ++++----
 arch/arm/boot/dts/rk3288-miqi.dts     | 12 ++++++++----
 arch/arm/boot/dts/rk3288-popmetal.dts |  8 ++++++++
 5 files changed, 30 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-fennec.dts b/arch/arm/boot/dts/rk3288-fennec.dts
index 1a68124cd776..f05ef49d8922 100644
--- a/arch/arm/boot/dts/rk3288-fennec.dts
+++ b/arch/arm/boot/dts/rk3288-fennec.dts
@@ -218,6 +218,10 @@
 	status = "okay";
 };
 
+&hevc_service {
+	status = "okay";
+};
+
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
@@ -511,7 +515,7 @@
 	status = "okay";
 };
 
-&vpu {
+&vpu_service {
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/rk3288-firefly.dtsi b/arch/arm/boot/dts/rk3288-firefly.dtsi
index 57aca1c19b67..c21c742efcb2 100644
--- a/arch/arm/boot/dts/rk3288-firefly.dtsi
+++ b/arch/arm/boot/dts/rk3288-firefly.dtsi
@@ -276,6 +276,10 @@
 	status = "okay";
 };
 
+&hevc_service {
+	status = "okay";
+};
+
 &i2c0 {
 	clock-frequency = <400000>;
 	status = "okay";
@@ -679,7 +683,7 @@
 	status = "okay";
 };
 
-&vpu {
+&vpu_service {
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/rk3288-miniarm.dts b/arch/arm/boot/dts/rk3288-miniarm.dts
index 4a3c26f4c6a9..069e7b2e1610 100644
--- a/arch/arm/boot/dts/rk3288-miniarm.dts
+++ b/arch/arm/boot/dts/rk3288-miniarm.dts
@@ -168,6 +168,10 @@
 	};
 };
 
+&hevc_service {
+	status = "okay";
+};
+
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
@@ -469,10 +473,6 @@
 	status = "okay";
 };
 
-&hevc_service {
-	status = "okay";
-};
-
 &vpu_service {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/rk3288-miqi.dts b/arch/arm/boot/dts/rk3288-miqi.dts
index 9aeb5fb83439..6de26c855e5c 100644
--- a/arch/arm/boot/dts/rk3288-miqi.dts
+++ b/arch/arm/boot/dts/rk3288-miqi.dts
@@ -206,6 +206,10 @@
 	status = "okay";
 };
 
+&hevc_service {
+	status = "okay";
+};
+
 &sdmmc {
 	supports-sd;
 	bus-width = <4>;
@@ -450,10 +454,6 @@ I2C
 	status = "okay";
 };
 
-&vpu {
-	status = "okay";
-};
-
 &usbphy {
 	status = "okay";
 };
@@ -482,6 +482,10 @@ I2C
 	status = "okay";
 };
 
+&vpu_service {
+	status = "okay";
+};
+
 &pinctrl {
 	pcfg_pull_none_drv_8ma: pcfg-pull-none-drv-8ma {
 		drive-strength = <8>;
diff --git a/arch/arm/boot/dts/rk3288-popmetal.dts b/arch/arm/boot/dts/rk3288-popmetal.dts
index 7475c295d392..376729cacf1a 100644
--- a/arch/arm/boot/dts/rk3288-popmetal.dts
+++ b/arch/arm/boot/dts/rk3288-popmetal.dts
@@ -193,6 +193,10 @@
 	status = "okay";
 };
 
+&hevc_service {
+	status = "okay";
+};
+
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
@@ -509,6 +513,10 @@
 	status = "okay";
 };
 
+&vpu_service {
+	status = "okay";
+};
+
 &uart0 {
 	status = "okay";
 };
-- 
2.35.3

